app/models/journeys/base.rb

```diff
@@ -3,7 +3,9 @@ module Base
     # TODO: move app/forms/*_forms to shared and journey specific folders
     # but needs load_paths sorting
     SHARED_FORMS = {
-      "current-school" => CurrentSchoolForm
+      "current-school" => CurrentSchoolForm,
+      # TODO: Not a shared form. This is journey specific, see TODO above.
+      "induction-completed" => InductionCompletedForm
```

Comment:
This can now be moved out into a const in the appropriate journey module, as a `FORMS` constant for each journey has been added https://github.com/DFE-Digital/claim-additional-payments-for-teaching/blob/0b46f2d8db3ea000738bc7f541aacba3c00ff168/app/models/journeys/additional_payments_for_teaching.rb#L12